packages/a11y/README.md

```diff
@@ -70,4 +70,10 @@ For context I’ll quote [this article on WordPress.org](https://make.wordpress.
 
 See <https://make.wordpress.org/design/handbook/design-guide/browser-support/>
 
+## Contributing to this package
+
+This package is part of the Gutenberg project. The repository is organized as a monorepo and contains many packages.
```

Comment:
We could also make it clear that this monorepo contains all WordPress packages used in the core. Maybe also link to https://make.wordpress.org/core/ in some way to make that connection explicit.